.github/workflows/release.yml

```diff
@@ -32,10 +32,29 @@ jobs:
         with:
           name: windows-artifacts
           path: /tmp/artifacts
-      - name: Set DRY_RUN_OPTION env variable
-        if: github.event.inputs.dry-run
-        run: echo "DRY_RUN_OPTION=--dry-run" >> $GITHUB_ENV
       - name: Deploy
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: dotnet run --project tools-shared/FakeItEasy.Deploy -- $DRY_RUN_OPTION -r "FakeItEasy/FakeItEasy.Analyzers" -t "${{ github.ref_name }}" -a "/tmp/artifacts/output"
+        run: |
+          if [[ "${{ github.event.inputs.dry-run }}" == "true" ]]; then
+            DRY_RUN_OPTION=--dry-run
+          fi
+          dotnet run --project tools-shared/FakeItEasy.Deploy -- $DRY_RUN_OPTION -r "${{ github.repository }}" -t "${{ github.ref_name }}" -a "/tmp/artifacts/output"
+
+  deploy-docs:
+    needs: deploy
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Trigger docs generation
+        run: |
+          publishMode=auto-merge
+          if [[ "${{ github.event.inputs.dry-run }}" == "true" ]]; then
+            publishMode=none

```

Comment:
This mode doesn't actually exist in `generate-docs.yml` yet, but I think it would make sense to add it. When running in dry-run mode, we don't want to create a PR nor merge the docs changes (obviously).

Reply:
(This is the reason the PR is WIP, btw. Calling the workflow with `none` will fail because it's not one of the possible choices)